src/components/Resource/ResourceDetails.tsx

```diff
@@ -53,321 +51,172 @@ export default function ResourceDetails(props: { id: string }) {
     );
   };
 
-  const ApprovalLetter = (data: any) => {
-    return (
-      <div id="section-to-print" className="print bg-white">
-        <div className="mx-4 p-2 lg:mx-20">
-          <div className="mt-6 text-center text-xl font-bold">
-            APPROVAL LETTER
-          </div>
-          <div className="mt-6 text-right">
-            <span className="font-semibold leading-relaxed">
-              {" "}
-              Date and Time:{" "}
-            </span>
-            {formatDateTime(data.created_date)}
+  if (loading || !data) {
+    return <Loading />;
+  }
+
+  return (
+    <Page
+      title={"Resource details"}
+      crumbsReplacements={{ [props.id]: { name: data.title } }}
+      backUrl={"/resource/board"}
+    >
+      <div className="mx-3 mb-10 md:mx-8">
+        <div className="my-4 flex flex-col items-start md:flex-row md:items-center md:justify-between">
+          <ButtonV2 href={`/resource/${props.id}/print`}>
+            <CareIcon icon="l-file-alt" className="mr-2 text-lg" /> Approval
+            Letter
+          </ButtonV2>
+        </div>
+        {data.assigned_to_object && (
+          <div className="relative rounded-lg bg-primary-200 shadow">
+            <div className="mx-auto max-w-screen-xl p-3 sm:px-6 lg:px-8">
+              <div className="pr-16 sm:px-16 sm:text-center">
+                <p className="font-bold text-primary-800">
+                  <span className="inline">
+                    Assigned to: {formatName(data.assigned_to_object)} -{" "}
```

Comment:
Let's add translations for all these.

Search for existing key-value pairs in en.json, if not add them, and then use useTranslation hook.